src/Symfony/Component/Routing/Router.php

```diff
@@ -329,18 +354,37 @@ public function addExpressionLanguageProvider(ExpressionFunctionProviderInterfac
     }
 
     /**
+     * This method is public because it needs to be callable from a closure in PHP 5.3. It should be converted back to protected in 3.0.
+     * @internal
      * @return GeneratorDumperInterface
      */
-    protected function getGeneratorDumperInstance()
+    public function getGeneratorDumperInstance()
```

Comment:
Not sure, http://symfony.com/doc/current/contributing/code/bc.html does not cover it (or I missed it).

Yes, it breaks subclasses that override this method as protected. No, as I don't see that it is `@api`. 

Probably something I could work around if needed.